cdap-standalone/src/test/java/co/cask/cdap/StandaloneTester.java

```diff
@@ -61,6 +62,16 @@ protected void before() throws Throwable {
     cConf.setBoolean(Constants.Dangerous.UNRECOVERABLE_RESET, true);
     cConf.setBoolean(StandaloneMain.DISABLE_UI, true);
 
+    // Setup test case specific configurations.
+    // The system properties are usually setup by TestConfiguration class using @ClassRule
+    for (String key : System.getProperties().stringPropertyNames()) {
+      if (key.startsWith(TestConfiguration.PROPERTY_PREFIX)) {
+        String value = System.getProperty(key);
+        cConf.set(key.substring(TestConfiguration.PROPERTY_PREFIX.length()), System.getProperty(key));
+        LOG.info("Custom configuration set: {} = {}", key, value);
```

Comment:
I think this will be seldomly used (i.e. for select confiugrations) that info is fine.
It's also info in TestBase, where I got this logic from (to attempt to also put the same functionality in StandaloneTester).
https://github.com/caskdata/cdap/blob/2b0fa17a72a508fa6aac0341fff8224609025b12/cdap-unit-test/src/main/java/co/cask/cdap/test/TestBase.java#L302-L308